R/pkg/R/functions.R

```diff
@@ -4854,7 +4855,8 @@ setMethod("current_timestamp",
           })
 
 #' @details
-#' \code{timestamp_seconds}: Creates timestamp from the number of seconds since UTC epoch.
+#' \code{timestamp_seconds}: Converts the number of seconds from unix epoch (1970-01-01 00:00:00 UTC)
+#' to a timestamp of that moment in the current system time zone.
```

Comment:
Precisely speaking, the current system time zone doesn't play any role in the conversion.

Reply:
True, I removed "of that moment in the current system time zone." from the docstrings.